nvflare/app_common/widgets/intime_model_selector.py

```diff
@@ -25,27 +25,30 @@
 
 
 class IntimeModelSelector(Widget):
-    def __init__(self, weigh_by_local_iter=False, aggregation_weights=None):
+    def __init__(
+        self, weigh_by_local_iter=False, aggregation_weights=None, validation_metric_name=MetaKey.INITIAL_METRICS
+    ):
         """Handler to determine if the model is globally best.
 
         Args:
             weigh_by_local_iter (bool, optional): whether the metrics should be weighted by trainer's iteration number.
             aggregation_weights (dict, optional): a mapping of client name to float for aggregation. Defaults to None.
+            validation_metric_name (str, optional): key used to save initial validation metric in the DXO meta properties (defaults to MetaKey.INITIAL_METRICS).
         """
         super().__init__()
 
         self.val_metric = self.best_val_metric = -np.inf
         self.weigh_by_local_iter = weigh_by_local_iter
-        self.validation_metric_name = MetaKey.INITIAL_METRICS
+        self.validation_metric_name = validation_metric_name
         self.aggregation_weights = aggregation_weights or {}
 
-        self.logger.debug(f"model selection weights control: {aggregation_weights}")
+        self.logger.info(f"model selection weights control: {aggregation_weights}")
         self._reset_stats()
 
     def handle_event(self, event_type: str, fl_ctx: FLContext):
         if event_type == EventType.START_RUN:
```

Comment:
Change to listen to AppEventType.ROUND_STARTED

Reply:
Thanks. Changed.